apollo-backend/src/main/java/io/logz/apollo/blockers/BlockerService.java

```diff
@@ -31,14 +39,21 @@
     private final BlockerInjectableCommons blockerInjectableCommons;
     private final Map<String, Class<? extends BlockerFunction>> blockerTypeNameBindings;
     private final Reflections reflections;
+    private final StackController stackController;
+    private final EnvironmentsStackController environmentsStackController;
+    private final ServicesStackController servicesStackController;
 
     @Inject
-    public BlockerService(BlockerDefinitionDao blockerDefinitionDao, BlockerInjectableCommons blockerInjectableCommons) {
+    public BlockerService(BlockerDefinitionDao blockerDefinitionDao, BlockerInjectableCommons blockerInjectableCommons,
```

Comment:
The whole point in blockerIngectableCommons is not to pass the Daos around, but set the functions in this ingectable object. Please leave this c'tor as it is